example.config.toml

```diff
@@ -1,6 +1,11 @@
+# All fields are required for now
 [general]
```

Comment:
Considering the code ignores this table anyway and having a general table probably doesn't make sense it can be removed